VoxelSniperCore/src/main/java/com/github/kevindagame/brush/CloneStampBrush.java

```diff
@@ -36,8 +32,8 @@ public class CloneStampBrush extends StampBrush {
     private void clone(final SnipeData v) {
         VoxelLocation point = getTargetBlock().getLocation().makeMutable();
         point.add(0, v.getcCen(), 0);
-        this.startingPoint = point.makeImmutable();
-        var positions = Shapes.cylinder(startingPoint, RotationAxis.Y, v.getBrushSize(), v.getVoxelHeight(), 0, false);
+        BaseLocation startingPoint = point.makeImmutable();
+        var positions = Shapes.cylinder(startingPoint, RotationAxis.Y, v.getBrushSize(), v.getVoxelHeight(), (v.getVoxelHeight() / 2.0) - 1, false);
```

Comment:
Am I correct in saying that this shifts the cylinder upwards so that the target center is now the bottom?

Reply:
If so, I wonder if there's value in making it configurable

Reply:
Yes, we shift it upwards. This can always be changed back using the vc command I think.